nsv13/code/modules/overmap/traders.dm

```diff
@@ -242,7 +242,22 @@
 			return
 		attempt_purchase(target, usr)
 	if(action == "mission")
-		give_mission(usr)
+		var/list/currentMissions = list()
+		for ( var/datum/nsv_mission/M in SSstar_system.all_missions )
```

Comment:
This will be weird on PVP. If, say, the astraeus crew cops all the 3 missions globally before NT

Cool idea though!